### ISSUE TITLE
Impaired declarative memory

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6773]
**Preferred term label:**
Declarative memory loss

**Synonyms**
Impaired declarative memory

**Definition (free text, please give PubMed ID)**
Impaired ability to remember facts and events.
PMID:33524264

**Parent term (use hpo.jax.org/app)**
Retrograde memory impairment

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
